specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2022-08-08/healthbot.json

```diff
@@ -0,0 +1,947 @@
+            "$ref": "./examples/ResourceUpdatePatch.json"
+          }
+        },
+        "description": "Patch a HealthBot.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/HealthBotUpdateParameters"
+            },
+            "description": "The parameters to provide for the required Azure Health Bot."
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is patched successfully, the service should return 200.",
+            "schema": {
+              "$ref": "#/definitions/HealthBot"
+            }
+          },
+          "201": {
+            "description": "If resource is received update request successfully, the service should return 201 (Created). Execution to continue asynchronously.",
+            "schema": {
+              "$ref": "#/definitions/HealthBot"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Bots_Delete",
+        "x-ms-examples": {
+          "BotDelete": {
+            "$ref": "./examples/ResourceDeletionDelete.json"
+          }
+        },
+        "description": "Delete a HealthBot.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/resourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "If resource is deleted successfully, the service should return 200."
+          },
+          "202": {
+            "description": "Accepted - Delete request accepted; operation will complete asynchronously"
+          },
+          "204": {
+            "description": "No content."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/Error"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.HealthBot/healthBots/{botName}/listSecrets": {
```

Comment:
>listSecrets [](http://example.com/codeflow?start=123&length=11)

If this is trying to get Secrets, please check "Secrets" section in [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)

Reply:
@zizw123  I added the `"x-ms-secret": true` annotation. Looking into the Secrets section, that's the only thing that was missing. Please let me know if anything else is needed.